code/index.html

```diff
@@ -1,33 +1,56 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./style.css" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
-    <title>Chat Bot</title>
-  </head>
-
-  <body>
-    <h1>Welcome to xxx.</h1>
-    <main>
-      <section class="chat" id="chat"></section>
-
-      <div class="input-wrapper" id="input-wrapper">
-        <form id="name-form">
-          <label for="name-input">Name</label>
-          <input id="name-input" type="text" />
-          <button class="send-btn" type="submit">
-            Send
-          </button>
-        </form>
-      </div>
-    </main>
-
-    <script src="./script.js"></script>
-  </body>
-
-</html>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+    rel="stylesheet" />
+  <title>Nosy bot</title>
+</head>
+
+<body>
+  <h1>Oh no! Here comes Nosy the bot!</h1>
+  <main>
+    <section class="chat" id="chat"></section>
+
+
+    <div class="buttonChoices">
+      <button class="morningButton" id="morningButton" type="submit">
+        Morning
+      </button>
+      <button class="nightButton" id="nightButton" type="submit">
+        Night
+      </button>
+      <button class="yesButton" id="yesButton" type="submit">
+        Yes
+      </button>
+      <button class="noButton" id="noButton" type="submit">
+        No
+      </button>
+      <button class="youngButton" id="youngButton" type="submit">
+        Young
+      </button>
+      <button class="oldButton" id="oldButton" type="submit">
+        Old
+      </button>
+    </div>
+
+    <div class="input-wrapper" id="input-wrapper">
+
+      <form id="name-form">
+        <label for="name-input"></label>
+        <input id="name-input" type="text" />
+        <button class="send-btn" id="oneButton" type="submit">
+          Send
+        </button>
+      </form>
+    </div>
+  </main>
+
+  <script src="./script.js"></script>
+</body>
```

Comment:
I keep getting this in the console of your chatbot, i dont know that it is?

<img width="631" alt="Screenshot 2023-09-04 at 19 08 37" src="https://github.com/Technigo/project-chatbot/assets/141672105/19a0a429-fca3-41f6-9495-bdb88c1c846d">